examples/kafka/kafka-ephemeral.yaml

```diff
@@ -22,6 +22,9 @@ spec:
       log.message.format.version: "2.6"
     storage:
       type: ephemeral
+    logging:
+      type: external
+      name: my-external-config-map  
```

Comment:
Why are you adding this? I do not think this should be in the examples - maybe you didn't planned to commit this?